### DISCUSSION TITLE
PlayerChunkUnloadEvent

### DISCUSSION THREAD

**Discussion by kylegrahammatzen** [Minestom/Minestom#1691] [Help] [answered]
Is there a plan within the community to handle the PlayerChunkUnloadEvent as a collection of chunks where the viewers are 0 and to be offloaded in batches by the developer's code?

Otherwise, I was looking into making my own InstanceContainers.

**Comment by kylegrahammatzen** [comment, marked as answer]
Hi all, just wondering if anyone has had a chance to review my discussion and has any ideas or feedback to share. It's been a week since I posted and I haven't received any responses yet. Any input would be greatly appreciated. Thanks!

**Reply by zax71** [reply]
Might get a better response on discord as this seams more of a discussion than a help question
